doc/integrator/ngeo.rst.tmpl

```diff
@@ -117,3 +99,14 @@ You also have a ``default`` entry from where we can give the base values for ``c
 The possible values are described in the ngeo documentation
 `ngeo constants <https://camptocamp.github.io/ngeo/${MAIN_BRANCH}/jsdoc/module-src_options.html>`_,
 `gmf constants <https://camptocamp.github.io/ngeo/${MAIN_BRANCH}/jsdoc/module-contribs_gmf_src_options.html>`_.
+
+CSS style
+---------
+
+In the ``gmfOptions`` we have a ``cssVars`` property where we can configure the CSS variable
```

Comment:
Which CSS variable?
Or do we mean here "all CSS variables"?

Reply:
=> all :-)